src/main/java/io/divolte/server/config/GoogleCloudStorageSinkConfiguration.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 GoDataDriven B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.divolte.server.config;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import javax.annotation.ParametersAreNullableByDefault;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+
+import io.divolte.server.filesinks.FileFlushingPool;
+import io.divolte.server.filesinks.FileManager.FileManagerFactory;
+import io.divolte.server.filesinks.gcs.GoogleCloudStorageFileManager;
+
+@ParametersAreNonnullByDefault
+public class GoogleCloudStorageSinkConfiguration extends FileSinkConfiguration {
+    public final String bucket;
+
+    @JsonCreator
+    @ParametersAreNullableByDefault
+    GoogleCloudStorageSinkConfiguration(final FileStrategyConfiguration fileStrategy, @JsonProperty(required = true) final String bucket) {
```

Comment:
That's the default. The `@JsonProperty(required = true)` is consistent with other config classes.